.travis.yml

```diff
@@ -3,11 +3,16 @@ language: julia
 os:
   - linux
   - osx
+  - windows
 julia:
   - 1.0
```

Comment:
@omus thoughts on testing all these versions? Or do you think LTS and most recent release would suffice?

Reply:
I think LTS, most recent release, and nightly is fine for most projects